packages/editor-ui/src/components/canvas/elements/handles/render-types/CanvasHandleNonMainInput.vue

```diff
@@ -55,7 +53,7 @@ function onClickAdd() {
 				v-if="isHandlePlusAvailable"
 				v-show="isHandlePlusVisible"
 				:handle-classes="handleClasses"
-				:type="plusType"
+				type="ai"
```

Comment:
why ai?

Reply:
For lack of a better name I guess. Got any suggestion?

Reply:
what does ai stand for here? not clear to me

Reply:
yes

Reply:
ah okay ai for the ai nodes, sorry I thought some UI thing

Reply:
Renamed to `secondary`. Should be clearer now. Thanks for the suggestion!